src/modules/controllers/task.controllers.js

```diff
@@ -0,0 +1,24 @@
+const Expenses = require("../../schema/taskSchema");
+
+module.exports.getAllExpenses = (req, res) => {
+  try {
+    Expenses.find().then((result) => {
+      res.send({ data: result });
+    });
+  } catch (e) {
+    res.status(500).send({ message: "Internal server error" });
+  }
+};
+
+module.exports.createNewExpense = (req, res) => {
+  try {
+    const newExpense = new Expenses(req.body);
+    newExpense.save().then(() => {
+      Expenses.find().then((result) => {
+        res.send({ data: result });
```

Comment:
what will the client get if an error occurs while creating the entry? use an error handler